src/instructlab/model/train.py

```diff
@@ -390,6 +388,9 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="By default, checkpoints are saved at the end of each training epoch. This option disables this behavior.",
 )
+@click.option(
+    "--pipeline", type=click.Choice(["simple", "full", "library"]), default="simple"
+)
```

Comment:
As we discussed in-person, should be 
```suggestion
@click.option(
    "--pipeline", type=click.Choice(["simple", "full", "accelerated"]), default="simple"
)
```